src/Codeception/Lib/Generator/Actions.php

```diff
@@ -113,7 +113,8 @@ protected function addMethod(\ReflectionMethod $refMethod)
         $methodTemplate = (new Template($this->methodTemplate))
             ->place('module', $module)
             ->place('method', $refMethod->name)
-            ->place('return_type', $this->createReturnTypeHint($refMethod))
+            ->place('return_type', $returnType = $this->createReturnTypeHint($refMethod))
```

Comment:
The patch looks good, however I really don't like this inline assignment, 
please extract this assignment to separate statement above `$methodTemplate = (new Template($this->methodTemplate))` line.

Reply:
No problem will do :+1:

Reply:
Updated 2b8d8aa